src/main/java/ch/njol/skript/classes/data/BukkitEventValues.java

```diff
@@ -616,6 +617,20 @@ public Entity get(final EntityTameEvent e) {
 			}
 		}, 0);
 
+		// EntityTeleportEvent
+		EventValues.registerEventValue(EntityTeleportEvent.class, Location.class, new Getter<Location, EntityTeleportEvent>() {
+			@Override
+			public @Nullable Location get(EntityTeleportEvent event) {
+				return event.getFrom();
+			}
+		}, EventValues.TIME_PAST);
+		EventValues.registerEventValue(EntityTeleportEvent.class, Location.class, new Getter<Location, EntityTeleportEvent>() {
+			@Override
+			public @Nullable Location get(EntityTeleportEvent event) {
+				return event.getTo();
+			}
+		}, EventValues.TIME_FUTURE);
```

Comment:
to allow the use of `event-location` instead of forcing `future event-location`, this will also allow future
```suggestion
		}, EventValues.TIME_NOW);
```